apisix/plugins/authz-keycloak.lua

```diff
@@ -16,11 +16,13 @@
 --
 local core      = require("apisix.core")
 local http      = require "resty.http"
+local secret    = require("apisix.secret")
```

Comment:
I'm not sure but importing just `fetch_secrets()` (instead of the whole module) might be more efficient. cc: @monkeyDluffy6017

Reply:
@Revolyssup please check this

Reply:
done